### ISSUE TITLE
Missing uploaded albums

### ISSUE THREAD

**Issue by forslund** [mopidy/mopidy-gmusic#141]
I'm missing two of my uploaded albums when listing them using mopidy-gmusic (1.0.0 is far as I can see)

I use the browse function to list available albums
```
curl -X POST -H Content-Type:application/json -d '{
  "method": "core.library.browse",
  "jsonrpc": "2.0",
  "params": {
    "uri": "gmusic:album"
  },
  "id": 1
}' http://localhost:6680/mopidy/rpc
```

This returns a list of 6 albums and the google web application and on my phone there's 8 albums.

What is the best way to test the latest development version?

**Comment by jjok** [comment]
I think this is fixed, although you may start seeing #139.

You can install the latest version by cloning this repository and installing.

On Debian:

```
sudo apt-get install git
sudo apt-get install python-dev
git clone https://github.com/mopidy/mopidy-gmusic
cd mopidy-gmusic
sudo pip install . --upgrade
```

**Comment by forslund** [comment]
Yes it seems to be fixed. Thanks for the help with the install.
